managed/services/checks/registry.go

```diff
@@ -36,20 +29,13 @@ type registry struct {
 	rw sync.RWMutex
 	// Results stored grouped by interval and by check name. It allows us to remove results for specific group.
 	checkResults map[check.Interval]map[string][]services.CheckResult
```

Comment:
Can we not store data in memory? We need to rewrite it for HA purposes.

Reply:
We can, but in scope of another ticket. This one just removes AM.